e2e-tests/fixtures/Sequence.ts

```diff
@@ -137,7 +137,7 @@ export class Sequence {
 
     expect(await this.linter.count()).toBe(0);
     await expect(this.command).toHaveText('C FSW_CMD_0 "ON" false 1');
-    await this.detailPane.getByRole('combobox').selectOption('OFF');
+    await this.page.locator('fieldset').filter({ hasText: 'ONOFF' }).getByRole('combobox').selectOption('OFF');
```

Comment:
Can this locator be a little more specific where rather than it looking for "ONOFF", can it look for the name of the argument?

`e.g. locator('fieldset').filter({ hasText: 'NAME OF ARGUMENT' }).getByRole('combobox')`